app/models/document_observer.rb

```diff
@@ -34,11 +34,9 @@ class DocumentObserver < ActiveRecord::Observer
 
 
   def after_create(document)
+    return unless Setting.notified_events.include?('document_added')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.